docs/docs/bloom-filter.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Bloom filter
 linkTitle: Bloom filter
-description: Bloom filters are a probabilistic data structure that checks for presence of an element in a set
+description: Bloom filter is a probabilistic data structure that checks for presence of an element in a set
```

Comment:
```suggestion
description: A Bloom filter is a probabilistic data structure that checks for presence of an element in a set
```

Reply:
No need. This is telegraphic writing and is aligned with all other data structures.